src/constants/tokenRestrictions.tsx

```diff
@@ -1,3 +1,4 @@
+import { isProductionEnv } from 'utils/env'
```

Comment:
Please run lint as Githubactions reports lint error:
"Run autofix to sort these imports!"